docs/deep-dive/03-dd-expose-function-serverless.md

```diff
@@ -21,6 +23,66 @@ This tutorial is based on an existing Function. To create one, follow the [Creat
 Follows these steps:
 
 <div tabs name="steps" group="expose-function">
+  <details>
+  <summary label="cli">
+  Kyma CLI
+  </summary>
+
+1. Export these variables:
+
+      ```bash
+      export DOMAIN={DOMAIN_NAME}
+      export NAME={APIRULE_NAME}
+      ```
+
+2. Download the latest configuration of the Function from the cluster. This way you will update the local `config.yaml` file with Function's code.
+
+  ```bash
+  kyma sync function $NAME -n $NAMESPACE
+  ```
+
+3. Edit the local `config.yaml` file and add the **apiRules** schema for the Function at the end of the file:
+
+  ```yaml
+  apiRules:
+      - name: {APIRULE_NAME}
+        service:
+          host: {APIRULE_NAME}.{DOMAIN_NAME}
+        rules:
+          - methods:
+              - GET
+              - POST
+              - PUT
+              - DELETE
+            accessStrategies:
+              - handler: noop
+  ```
+
+4. Apply the new configuration to the cluster:
+
+  ```bash
+  kyma apply function
+  ```
+
+5. Check if the Function's code was pushed to the cluster and reflects the local configuration:
+
+  ```bash
+  kubectl get apirules $NAME -n $NAMESPACE
+  ```
+
+6. Check that the API Rule was created successfully and has the status `OK`:
+
+    ```bash
```

Comment:
Please fix the indentation: 

<img width="769" alt="Screen Shot 2021-05-21 at 14 13 17" src="https://user-images.githubusercontent.com/35192450/119135472-bb6aa800-ba3e-11eb-8691-82c3c05e24b1.png">